src/services/web3/web3.plugin.ts

```diff
@@ -176,6 +190,12 @@ export default {
         // for when user reloads the app on an already connected wallet
         // need to store address to pre-load that connection
         if (account.value) {
+          // fetch sanctioned status
+          const _isSanctioned = await isSanctionedAddress(account.value);
+          isSanctioned.value = _isSanctioned;
+          if (_isSanctioned) {
+            disconnectWallet();
```

Comment:
may be add here `return statement`? 
now function disconnects users and then set all statements to `connected`